lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1338,8 +1338,17 @@ Error ONNXModelLoader::loadConvTranspose(const ONNX_NAMESPACE::NodeProto &op,
   }
 
   unsigned_t dilation = 1;
-  if (dict.count("dilation")) {
-    ASSIGN_VALUE_OR_RETURN_ERR(dilation, loadInt(dict.at("dilation")));
+  if (dict.count("dilations")) {
+    std::vector<unsigned_t> dilations(2, 1);
```

Comment:
tiny nit: Unnecessary/a little misleading to initialize this, as it will be replaced by getShape in the next line.

Reply:
Done.